src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -387,9 +387,9 @@ public static string SetParameterDefaultStateFrom(string parameterDisplayName, b
             return $"Updating default state for '{parameterDisplayName}' from {oldIsDefault} to {newIsDefault}";
          }
 
-         public static string LoadProjectFromSnapshotDescription(string snapshotFile, string version)
+         public static string LoadProjectFromSnapshotDescription(string snapshotFile, string version, string snapshotVersion)
          {
-            return $"Project loaded from snapshot file '{snapshotFile}' with version {version}";
+            return $"Project loaded from snapshot file '{snapshotFile}' which was created with version {snapshotVersion}. Project version is now {version}.";
```

Comment:
Also now that we have the snapshot version, we can display it as requested. I am not sure how you want it tho. Please make another issue if you would like something else